corehq/apps/app_manager/views/download.py

```diff
@@ -293,7 +295,8 @@ def create_build_files_if_necessary_handling_conflicts(is_retry=False):
             payload = convert_XML_To_J2ME(payload, path, request.app.use_j2me_endpoint)
         response.write(payload)
         if path in ['profile.ccpr', 'media_profile.ccpr'] and request.app.last_released:
-            response['X-CommCareHQ-AppReleasedOn'] = request.app.last_released.isoformat()
+            last_released = ServerTime(request.app.last_released).user_time(pytz.UTC).done().isoformat()
```

Comment:
does this need `if request.app.last_released`?

Reply:
Not following - the line above does check `request.app.last_released`